### ISSUE TITLE
Segmentation fault in stage 1: test logits

### ISSUE THREAD

**Issue by LAJ-THU** [martinpacesa/BindCraft#223]
I followed the installation guide and ran the test case given (PDL1) with :
`python -u ./bindcraft.py --settings './settings_target/PDL1.json' --filters './settings_filters/default_filters.json' --advanced './settings_advanced/default_4stage_multimer.json'`

However, I encountered a segmentation fault. I don't know why and how to solve it.

**Comment by martinpacesa** [comment]
Is this a local installation?

**Comment by LAJ-THU** [comment]
Yes I installed BindCraft in an old machine. My glibc version is too old, which cannot support the latest pyrosetta so I downgraded my pyrosetta version to `2024.39+release.59628fbc5bc09f1221e1642f1f8d157ce49b1410 2024-09-23` and this version encountered segmentation fault. 

thanks for your reply

**Comment by martinpacesa** [comment]
Yeah this happens due to a mismatch between JAX and pyrosetta version, it is very system dependent, so I cannot help much there but you can search through the issues here, it has been mentioned several times
